### PR TITLE
Newsweek feature

### PR DESCRIPTION
**Pull request by henatan99**
Pull request for Newsweek page bootstrap clone project, Micorverse. 

In this project,  we were tasked to create a responsive clone for the Newsweek  Magazine homepage, using specifically Bootstrap CSS framework with some Bootstrap Techniques like bootstrap flexbox, bootstrap grid, system, etc.
The project has the following features:

**Mobile View** :
- Navigation bar with a logo, subscribe button, a user log in, and menu tab.
- Main section with bootstrap flex and grid items
- footer with footer nav list items and site logo plus copyright info.

**Tablet View** :
- Navigation bar with a logo, subscribe button, a user log in, and menu tab.
- Main section with bootstrap flex and grid items
- footer with footer nav list items and site logo plus copyright info.

**Laptop  View** :
- Navigation bar with a date, logo, subscribe button, a login, at the top, two nav-list items on above the other.
- Main section with bootstrap flex and grid items
- footer with footer nav list items and site logo plus copyright info

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1491 @@
...
+                <a href="#"> Advertise</a>
+                <a href="#">Copyright</a>
+                <a href="#">Terms & Conditions</a>
+                <a href="#">Privacy Policy</a>
+                <a href="#"> Cookie Policy</a>
+                <a href="#">Terms of Sale</a>
+                <a href="#">Archive</a>
+                <a href="#">Announcements</a>
+            </div>
+        </footer>
```
Comment (arslanbisharat) [inline index.html]: - [x] Please move footer section outside main section

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1491 @@
...
+        </nav>
+    </header>
+    <main class="container-fluid mt-3 main">
+        <div class="sections-wrapper-1 row">
+            <section class="top-story col-12 col-md-7 col-lg-5 order-md-2 pt-lg-1 mb-lg-0">
+                <h1 class="text-muted h6 font-weight-bold">TOP STORY</h1>
+                <article>
+                    <figure class="position-relative">
+                        <a href="#">
+                            <img src="assets/images/top-story/nancy-pelosi.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1491 @@
...
+                    </p>
+                </article>
+            </section>
+            <section class="featured-stories col-12 col-md-5 col-lg-3 order-md-1">
+                <div class="mr-lg-n5 p-lg-2">
+                    <h1 class="text-muted h6 font-weight-bold">FEATURED STORIES</h1>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/trump.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1491 @@
...
+                        <p>
+                            Trump told his supporters in Des Moines, Iowa that he rated the
+                            mental state of the three autocratic rulers above that of his Democratic rival,
+                            Joe Biden.
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="">
+                                <img src="assets/images/featured-section/greece-turkey-military-dispute-mediterranean.jpg"
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h1>
+                        <p>
+                            The president's comments come amid an ongoing stalemate over the details
+                            of a further relief package amid the coronavirus crisis.
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/judge-amy-coney-barrett.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h1>
+                        <p>
+                            The Democratic challenger has opened up a one-point lead over the
+                            Republican incumbent with just weeks to go until polling day.
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/russia-ryabkov-nuclear-weapons-talks.jpg"
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1491 @@
...
+                        </article>
+                        <hr style="height:1px;border:none;color:#333;background-color:#333;">
+                    </section>
+                    <section class="opinion mt-4 mb-4">
+                        <h2>
+                            <a href="#" class="text-muted">OPINION</a>
+                        </h2>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/bruce-abramson-jeff-ballabon.png" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,1491 @@
...
+                                    <a href="#">How Dare Republicans Tout Barrett's Motherhood While Letting Working
+                                        Moms Drown?</a>
+                                </h2>
+                                <h2 class="text-muted">BY JENNIFER WEISS-WOLF</h2>
+                            </div>
+                        </article>
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/david-p-goldman.png" alt="" style="padding: 0;">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,1491 @@
...
+                                <h2>
+                                    <a href="#">Biden's 'Unconstitutional' Game</a>
+                                </h2>
+                                <h2 class="text-muted">BY MIKE DAVIS</h2>
+                            </div>
+                        </article>
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/harry-j-kazianis.png" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,1491 @@
...
+                                <h2>
+                                    <a href="#">Huawei Plays Matador to Trump's Bull</a>
+                                </h2>
+                                <h2 class="text-muted">BY DAVID P. GOLDMAN</h2>
+                            </div>
+                        </article>
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/jennifer-weiss-wolf.png" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,1491 @@
...
+                                <h2>
+                                    <a href="#">Why Won't Joe Biden Repudiate Anti-Semitic Democrats?</a>
+                                </h2>
+                                <h2 class="text-muted">BY BRUCE ABRAMSON AND JEFF BALLABON</h2>
+                            </div>
+                        </article>
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/mike-davis.png" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,1491 @@
...
+                                <h2>
+                                    <a href="#">What We Must Learn from the COVID-19 Response</a>
+                                </h2>
+                                <h2 class="text-muted">BY JAY W. RICHARDS, DOUGLAS AXE AND WILLIAM BRIGGS</h2>
+                            </div>
+                        </article>
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/richards-axe-briggs.png" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,1491 @@
...
+                                <h2>
+                                    <a href="#">North Korea's New ICBM: A 21st-Century Spruce Goose?</a>
+                                </h2>
+                                <h2 class="text-muted">BY HARRY J. KAZIANIS</h2>
+                            </div>
+                        </article>
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/robert-reich.png" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,1491 @@
...
+                                <h2>
+                                    <a href="#">Vibes in the Middle East</a>
+                                </h2>
+                                <h2 class="text-muted">BY SHOSHANA BRYEN</h2>
+                            </div>
+                        </article>
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/shoshana-bryen.png" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 15]
```diff
@@ -0,0 +1,1491 @@
...
+                        </article>
+                    </section>
+                </div>
+            </div>
+            <section class="featured-sponored col-12 col-md-5 col-lg-3 order-md-2 order-lg-4">
+                <div class="mr-lg-n5 p-lg-2">
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/holy-angel-university.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 16]
```diff
@@ -0,0 +1,1491 @@
...
+                        <p class="p-3">
+                            What does it take to be a leader in Global Education?
+                            What sets the best Universities apart from the others on the world stage?
+                            Find out more.
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/americas-best-banks-2021.png" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 17]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h1>
+                        <p class="p-3">
+                            Introducing Newsweek's first ranking of the financial institutions
+                            that best serve their customers' needs in today's challenging times.
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/worlds-best-specialized-hospitals-2021.png"
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 18]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h1>
+                        <p class="p-3">
+                            In the first part of this special,
+                            we rank the best hospitals in cardiology, oncology and endocrinology.
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/best-online-shops-2021.png" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 19]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h1>
+                        <p class="p-3">
+                            Newsweek and Statista are recognizing 1,000 shops across 39 categories in
+                            eight groups.
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/americas-best-physical-rehabilitation-centers-2020.png"
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 20]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h1>
+                        <p class="p-3">
+                            Finding reliable rehabilitation can be a challenge. We partnered with
+                            Statista to rank the country's Best Physical Rehabilitation Centers.
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/brandspark.png" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 21]
```diff
@@ -0,0 +1,1491 @@
...
+            </section>
+            <div class="middle-sections-wrapper col-12 col-md-7 col-lg-5 order-md-4 order-lg-5 ">
+                <section class="my-turn mb-4">
+                    <h1>
+                        <a href="#" class="text-muted">MY TURN</a>
+                    </h1>
+                    <div class="row">
+                        <article class="col-12 col-md-4 mr-n2 ">
+                            <a href="#">
+                                <img src="assets/images/my-turn/doctors-black-doctors-black-women.jpg" alt=""
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 22]
```diff
@@ -0,0 +1,1491 @@
...
+                                    class="mb-3">
+                            </a>
+                            <h2>
+                                <a href="#" class="text-muted">'I'm a Black Doctor. I Feel the Weight of Representing My
+                                    Community'</a>
+                            </h2>
+                        </article>
+                        <article class="col-12 col-md-4 ml-n1 mr-n1">
+                            <a href="#">
+                                <img src="assets/images/my-turn/chef-cooking-tiktok-pandemic-coronavirus.png" alt=""
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 23]
```diff
@@ -0,0 +1,1491 @@
...
+                                    class="mb-3">
+                            </a>
+                            <h2>
+                                <a href="#" class="text-muted">'I Was Laid Off in March, Now I'm Making Money As A
+                                    TikTok Cooking Star'</a>
+                            </h2>
+                        </article>
+                        <article class="col-12 col-md-4 ml-n2">
+                            <a href="#">
+                                <img src="assets/images/my-turn/love-romance-italy-romeo-juliet-pandemic.jpg" alt=""
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 24]
```diff
@@ -0,0 +1,1491 @@
...
+                </section>
+                <section class="culture & travel mb-4">
+                    <h1>
+                        <a href="#" class="text-muted">CULTURE & TRAVEL</a>
+                    </h1>
+                    <div class="row">
+                        <article class="col-12 col-md-4 mr-n2 ">
+                            <figure>
+                                <a href="#">
+                                    <img src="assets/images/culture-and-travel/singapore-airlines.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 25]
```diff
@@ -0,0 +1,1491 @@
...
+                            </figure>
+                            <h2>
+                                <a href="#" class="text-muted">Missing Flying? Flights to Nowhere Take Off As a New
+                                    Travel Trend</a>
+                            </h2>
+                        </article>
+                        <article class="col-12 col-md-4 ml-n1 mr-n1">
+                            <figure>
+                                <a href="#">
+                                    <img src="assets/images/culture-and-travel/100-best-rock-albums-all-time-according-critics.png"
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 26]
```diff
@@ -0,0 +1,1491 @@
...
+                            </figure>
+                            <h2>
+                                <a href="#" class="text-muted">100 Best Rock Albums of All Time, According to
+                                    Critics</a>
+                            </h2>
+                        </article>
+                        <article class="col-12 col-md-4 ml-n2">
+                            <figure>
+                                <a href="#">
+                                    <img src="assets/images/culture-and-travel/cul-map-wes-anderson-white-cyclon-banner.jpg"
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 27]
```diff
@@ -0,0 +1,1491 @@
...
+                            </h2>
+                        </article>
+                    </div>
+                </section>
+                <section class="more-stories position-sticky sticky-top">
+                    <h1 class="text-muted">MORE STORIES</h1>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/downtown-broadway-tennessee-coronavirus-august-2020.jpg"
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 28]
```diff
@@ -0,0 +1,1491 @@
...
+                        <p>
+                            "Dare anyone represent Facebook to discuss with me point-to-point in
+                            a live broadcast, based on my two reports on the lab-origin of COVID-19
+                            by CCP regime," Li-Meng Yan's account tweeted this week.
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/houston-ballot-drop-off.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 29]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h2>
+                        <p>
+                            The Republican senator declined to attend
+                            the debate, leaving his Libertarian opponent alone on stage.
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/mages/more-stories/lindsey-graham.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 30]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h2>
+                        <p>
+                            "This event continues to be very high risk," space debris
+                            tracking company LeoLabs said of the possible collision.
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/protest-chicago.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 31]
```diff
@@ -0,0 +1,1491 @@
...
+                        <p>
+                            Young, Black conservatives are not only a force to be reckoned with,
+                            they could be President Donald Trump's secret weapon. Here are eight
+                            of them who should make Joe Biden's supporters nervous.
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/elderly-mask.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 32]
```diff
@@ -0,0 +1,1491 @@
...
+                        <p>
+                            The White House coronavirus task force member said: "That's sort of like
+                            saying somebody was speeding in a car at 95 miles an hour and didn't get
+                            in an accident, so I can go ahead and speed and not get in an accident."
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/masks-coronavirus-covid-getty-stock.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 33]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h2>
+                        <p>
+                            Sir Iain Duncan Smith believes that the SAGE advisory group is
+                            unbalanced and scared of having unpopular opinions.
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/china-xi-jinping.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 34]
```diff
@@ -0,0 +1,1491 @@
...
+                        <p>
+                            The Chinese Communist Party chairman was speaking at a commemorative event
+                            in Shenzhen, Guangdong, China, surrounded by other maskless government
+                            officials including Hong Kong leader Carrie Lam.
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/protest-chicago.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 35]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h2>
+                        <p>
+                            The former vice president announced that his campaign raised a
+                            record breaking $383 million in September alone.
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/meghan-markle-admiralty-house-sydney-australia.jpg"
```
Comment (arslanbisharat) [inline index.html:355]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 36]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h2>
+                        <p>
+                            Tokyo and Beijing remain locked in a dispute over an uninhabited archipelago
+                            in the East China Sea which may be home to oil and natural gas reserves.
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/vladimir-putin.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 37]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h2>
+                        <p>
+                            Politicians and activists shared birthday messages on social media while
+                            people gathered at the memorial in Minneapolis on Wednesday.
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/former-secretary-state-hillary-clinton.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 38]
```diff
@@ -0,0 +1,1491 @@
...
+                        <p>
+                            The social media sites both limited the spread of an article that claimed
+                            Joe Biden's son, Hunter, had accepted money from a Ukrainian energy company
+                            in exchange for access.
+                        </p>
+                    </article>
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/us-president-donald-trump.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 39]
```diff
@@ -0,0 +1,1491 @@
...
+                        </p>
+                    </article>
+                </section>
+            </div>
+            <section class="ranking-and-sponsored col-12 col-lg-3 order-md-6 pl-0">
+                <div class="ml-lg-n5 position-sticky sticky-top mt-5">
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/rankings-and-sponored/americas-best-addiction-treatment-centers-2020.png"
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 40]
```diff
@@ -0,0 +1,1491 @@
...
+                        <p>
+                            In partnership with Statista, we established a ranking providing
+                            information and insights to help you during this challenging time.
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/rankings-and-sponored/americas-best-nursing-homes-2020.png"
+                                    alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 41]
```diff
@@ -0,0 +1,1491 @@
...
+                        </h1>
+                        <p>
+                            The best Art, Design and Architecture schools use a recipe consisting of a blend of
+                            ingredients. Read more here..
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/rankings-and-sponored/gravely-pc.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 42]
```diff
@@ -0,0 +1,1491 @@
...
+                                This Startup Is Making Long Distance Real Estate Investing Possible for Millennials
+                            </a>
+                        </h1>
+                        <p>75% of Roofstock users are first-time real estate investors, and more than half are under 35.
+                        </p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/rankings-and-sponored/image-3-this-startup-making.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

File: index.html [hunk 43]
```diff
@@ -0,0 +1,1491 @@
...
+                            <a href="#">
+                                What Makes the Best Lawyers Stand Out?
+                            </a>
+                        </h1>
+                        <p>IThe unthinkable has happened and you need a good lawyer to turn to, how do you chose?</p>
+                    </article>
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/rankings-and-sponored/savannah-college-art-design.jpg" alt="">
```
Comment (arslanbisharat) [inline index.html]: - [ ] Please use text for the **alt** attribute.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

                                      ### Project Specific 

- [x]  Please make your PR descriptive. Please add more details to it. Please refer to [link ](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642) from the pathway to learn more. 

- [x] Your CSS has too many font-size declarations (12), abstraction needed.

- [ ] bootstrap_classes_requirement

- [x]  Please use dry CSS wherever needed. I have given some hints through inline-comments below. Please work on it. 

- [ ]  In Bootstrap projects make sure to less use the styles.css file for styles that could be done using Bootstrap classes since that's the main goal of the exercise. Things like font-weight, flex justifications, display, etc could be implemented completely with Bootstrap classes. [Please refer to the main page of bootstrap to learn more about bootstrap. ](https://getbootstrap.com/docs/4.5/getting-started/introduction/). I have given some hints through inline-comments below. Please work on it. 

                                               ### Large Screen

- [x] In the navbar doesn't look like the original one. Please have a look below for suggestions. 

**Clone**

chrome-capture (7)

- [x] There should no white space in this section.

**clone**

chrome-capture (8)

- [x] This section doesn't look like the original one. 

**clone**

chrome-capture (9)

- [x] There should less space. 

**clone**

chrome-capture (10)

- [x] There should total of 8 items in **EDITOR'S PICK** but you have only 7. The heading of EDITOR'S PICK doesn't look like the original one. 

- [x] You have extra space in all of these sections please remove that. 

chrome-capture (11)

- [ ] Subscribe section should have more width than the newsletter section. n newsletter there should space between email input and signup button. 
chrome-capture (12)

- [ ] chrome-capture (13)

- [ ] Please male Edition bold, and add proper spacing and | after each item. 

**clone**

chrome-capture (14)


                                             ### MEDIUM SCREEN

- [ ] Please male Edition bold, and add proper spacing and | after each item. 

**clone**

chrome-capture (14)

                                         ###SMALL SCREEN

- [x] On the small screen, starting from the center of the page, the content is neither properly aligned nor has proper spacing. There is an overlapping of text. Please fix that. 



**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: index.html (hunk 1 above)
Comment (henatan99) [reply index.html]: footer section out of main section

File: README.md [hunk 44]
```diff
@@ -1,2 +1,71 @@
-# NewsWeek-page-bootstrap-clone
-This project is about building a replica of NewsWeek page using bootstrap framework and making it responsive. 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HTML form page
```
Comment (arslanbisharat) [inline README.md:3]: remove this

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [ ] On the large screen, the webpage has white space around the borders. Please work on it 

- [ ]  In the navbar doesn't look like the original one. Please have a look below for suggestions.
chrome-capture (25)

- [ ] Please remove white space from here by adding more items
img

- [ ] Newsletter has more image height than the subscribe section. 

chrome-capture (26)

- [ ] On tablet mode text of the items present in the center is present vertically. Please work on it. 

chrome-capture (28)


- [ ] On small screen image should be in the center 
chrome-capture (27)


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Requested Changes

- [ ] Please make changes asked in previous reviews. You have not made a single commit after your second review. 
In case of any question please reach me out on slack @arslanbisharat

**Review by arslanbisharat — APPROVED** [review]
#  Approved

Thank you for making those changes requested by previous TSEs.
You can merge and move forward with your curriculum.

Regards,
Arslanbisharat

**Pull request merged**